_config.yml

```diff
@@ -18,19 +18,24 @@ redcarpet:
   - with_toc_data
 
 versions:
-  stable: v1.0
-  dev: v1.1
+  stable: v1.1
+  # dev: v1.2
```

Comment:
note that without _any_ dev docs, the various links in the cockroach repo are now broken, causing tests to fail

Reply:
Yuck, thanks, @tamird. Something to remember for next time. Alex merged a fix just a bit ago.